packages/gatsby-kontent-components/src/image-element/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { GatsbyImageProps, GatsbyImage } from 'gatsby-plugin-image';
+import {
+  getGatsbyImageData,
+  GetGatsbyImageDataProps,
+} from './get-gatsby-image-data';
+
+export interface ImageItem {
+  image_id: string;
+  url: string;
+  description?: string;
+  name?: string;
```

Comment:
I guess `name` is not part of inline images in the delivery response. This property if retrieved only in Asset elements.

Reply:
Yep, that's correct but this property is optional. Nevertheless, I consider this interface as an envelope for `gatsby-plugin-image` therefore IMO it's ok that the `name` property is not in the delivery response. I reckon the user can choose the proper name according to the context of the element within image resolution.